### ISSUE TITLE
UnhandledPromiseRejectionWarning

### ISSUE THREAD

**Issue by hrqmonteiro** [windicss/windicss#767]
Trying to set it to Next.js and Webpack, and i literally copied what is saying [here](https://windicss.org/integrations/webpack.html#next-js) and this [example](https://github.com/windicss/windicss-webpack-plugin/tree/main/example/next)

And i get this error:

```
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
(node:484838) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/home/minutrade/vimprogramar/node_modules/windicss-webpack-plugin/dist/core/client.js'
    at Object.openSync (fs.js:498:3)
    at Object.readFileSync (fs.js:394:35)
    at Object.devtoolsLoader (/home/minutrade/vimprogramar/node_modules/windicss-webpack-plugin/dist/loaders/dev-tools.cjs:32:30)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:484838) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:484838) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

When i try to run it.

**Comment by await-ovo** [comment]
maybe related to https://github.com/windicss/windicss-webpack-plugin/issues/113

**Comment by harlan-zw** [comment]
Fixed, sorry about that @hrqmonteiro 

This is fixed in the webpack plugin v 1.7.3
